react/ProductSummary.js

```diff
@@ -40,60 +44,68 @@ class ProductSummary extends Component {
       showButtonOnHover,
     } = this.props
 
+    const product = this.props.product || createProduct()
+
     return (
-      <div className="tc pointer"
-        onMouseEnter={this.handleMouseEnter}
-        onMouseLeave={this.handleMouseLeave}>
-        {!product &&
-          <div>
-            <FormattedMessage id="loading" />
-          </div>
-        }
-        {product && (
-          <div>
-            <div onClick={this.handleClick}>
+      <div className="vtex-product-summary">
+        <Card fullWidth>
+          <div className="tc pointer"
+            onMouseEnter={this.handleMouseEnter}
+            onMouseLeave={this.handleMouseLeave}>
+            {!product &&
+            <div>
+              <FormattedMessage id="loading" />
+            </div>
+            }
+            {product && (
               <div>
-                {
-                  showBadge ? (
-                    <DiscountBadge
+                <div onClick={this.handleClick}>
+                  <div>
+                    {
+                      showBadge ? (
+                        <DiscountBadge
+                          listPrice={product.listPrice}
+                          sellingPrice={product.sellingPrice}
+                          label={badgeText}>
+                          <img className="vtex-product-summary__image" alt={product.name} src={product.imageUrl} />
+                        </DiscountBadge>
+                      ) : (
+                        <img className="vtex-product-summary__image" alt={product.name} src={product.imageUrl} />
+                      )
+                    }
+                  </div>
+                  <div className="vtex-product-summary__name-container pv5 f4 gray db tc">
+                    <ProductName
+                      name={product.name}
+                      skuName={product.skuName}
+                      brandName={product.brandName}
+                      referenceCode={product.referenceCode} />
+                  </div>
+                  <div className="vtex-price-container pv1">
+                    <Price
                       listPrice={product.listPrice}
                       sellingPrice={product.sellingPrice}
-                      label={badgeText}>
-                      <img alt={product.name} src={product.imageUrl} />
-                    </DiscountBadge>
-                  ) : (
-                    <img alt={product.name} src={product.imageUrl} />
-                  )
-                }
+                      installments={product.installments}
+                      installmentPrice={product.installmentPrice}
+                      showListPrice={showListPrice}
+                      showLabels={showLabels}
+                      showInstallments={showInstallments} />
+                  </div>
+                </div>
+                <div className="pv2">
+                  <div className={`${(!showButtonOnHover || (showButtonOnHover && this.state.isHovering)) ? 'db' : 'dn'}`}>
```

Comment:
Essa expressão pode ser simplificada para `!showButtonOnHover || this.state.isHovering`

Reply:
Tinha sido simplificada no PR do botão de detalhes, mas foi fechado :cry: